### PR TITLE
put husky@4.3.8 back

### PR DESCRIPTION
**Pull request by DrShpongle**

### CODE REVIEW

File: .husky/_/husky.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```
Comment (jbranchaud) [inline .husky/_/husky.sh:1]: I think this is from the v5 upgrade. This file should get deleted.

Comment (DrShpongle) [reply .husky/_/husky.sh:1]: oh, mixed it up with the one under .git folder!

Comment (DrShpongle) [reply .husky/_/husky.sh]: fixed

**Review by jbranchaud — APPROVED** [review]
Thanks for digging into this 🙌

**Pull request merged**